src/components/Form/DocumentSelection.jsx

```diff
@@ -20,14 +20,15 @@ import { docTypes } from '../../utils';
 const DocumentSelection = ({ htmlForAndIdProp, handleDocType, docType }) => (
   <Box sx={{ marginTop: '20px' }}>
     <FormControl sx={{ width: 220 }}>
-      <InputLabel id={`${htmlForAndIdProp}-label`}>Select Document Type</InputLabel>
+      <InputLabel id={`${htmlForAndIdProp}-label`}>Select Document Type *</InputLabel>
```

Comment:
Think you could just add required for `<InputLabel>` for the same effect

Reply:
Better yet, we can just put required from Line 31 from `<Select>` to`<FormControl>` in Line 22.

Reply:
The required in `<Select>` is already covered by `<FormControl>` in Line 22. That can be removed without breaking anything. It'll still achieve the same result.